README.rst

```diff
@@ -448,6 +448,62 @@ SERVERS
     to make multihost prototype support easier to verify as many current
     tests were developed with Restraint.
 
+Plugin options
+--------------
+
+.. note::
+
+   The following applies to situations when a plugin is specified
+   on the command line only. Keys of plugins specified in fmf files
+   would not be modified. This is a limit of the current implementation,
+   and will be addressed in the future.
```

Comment:
This is comment is rendered as:

    # Here the verbosity will not be increased since the plugin is # not mentioned on the

Nested verbatim block should help:

```suggestion
   and will be addressed in the future::
```

Reply:
Fixed in e79f6f2b.